CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ This release removes all of the old clustering code. It operates as a standalone
 - [#6178](https://github.com/influxdata/influxdb/issues/6178): Ensure SHARD DURATION is checked when recreating a retention policy
 - [#6223](https://github.com/influxdata/influxdb/issues/6223): Failure to start/run on Windows. Thanks @mvadu
 - [#6237](https://github.com/influxdata/influxdb/issues/6237): Enable continuous integration testing on Windows platform via AppVeyor. Thanks @mvadu
+- [#6257](https://github.com/influxdata/influxdb/issues/6257): CreateShardGroup was incrementing meta data index even when it was idempotent.
```

Comment:
0.12.0 has already been released. This should go into 0.13.0 and then be copied to 0.12.1 if it's going to be part of that release.